changelogs/fragments/2057-zos_data_set&data_set-Ensure-Dataset-Volume-Validation.yml

```
@@ -0,0 +1,8 @@
+bugfixes:
+   - zos_data_set - Added validation to check existing cataloged volumes 
```

We need to keep bugfixes entry in the following format https://docs.ansible.com/ansible/latest/community/development_process.html#creating-changelog-fragments 

```
Fixes that resolve issues. SHOULD not be used for minor enhancements (use minor_change instead). Use past tense to describe the problem and present tense to describe the fix.
```

```suggestion
   - zos_data_set - Attempting to create a data set with the same name on a different volume did not work, nor did it report a failure. The fix now informs the user that if the data set is cataloged on a different volume and it needs to be uncataloged before using the data set module to create a new data set on a different volume.
```

---

Resolved